uint8 joint0
uint8 joint1
uint8 joint2
uint8 joint3
uint8 joint4
uint8 gripper
